### DISCUSSION TITLE
Questions regarding version 3.0.10

### DISCUSSION THREAD

**Discussion by icrmrd** [nilsteampassnet/TeamPass#3941] [General]
Hi,
The latest release described seems to be 3.0.9 but the app available is actually on 3.0.10 already. I suppose the docs haven't been updated yet, right?
I'm concerned about the bugs and vulnerabilities that are described as solved after 3.0.9, so I'd like to confirm the current version and get feedbacks on how stable it is - any take? 
I also wonder if there are other requirements regarding the requirements (Mariadb, PHP + extensions, MySQL).

**Comment by nilsteampassnet** [comment]
@icrmrd 
Thank you for your question.
3.0.10 is net yet released.
It will soon as I want this version to become a "reference" because it will be the latest with PHP version 7.4 compliant.
Currently, you can use latest commit from master branch.
